### ISSUE TITLE
As a special privileged Rat I want to be able to view previous versions of the paperwork I'm viewing.

### ISSUE THREAD

**Issue by UncleClapton** [FuelRats/fuelrats.com#346]
## Acceptance Criteria

- [ ] 

## Additional Information

Blocked by: [API-398](https://jira.fuelrats.com/browse/API-398)

**Comment by SuperManifolds** [comment]
Won't do, too technically complex
